app/cafe.py

```diff
@@ -0,0 +1,18 @@
+from datetime import date
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor.keys():

```

Comment:
You can check directly `if "vaccine" not in visitor`, though I'm not sure it should be changed, since `visitor.keys()` is clearer.